config/config.go

```diff
@@ -392,11 +399,19 @@ func (sc *SamplerConfig) NewSampler(
 		options := []jaeger.SamplerOption{
 			jaeger.SamplerOptions.Metrics(metrics),
 			jaeger.SamplerOptions.InitialSampler(initSampler),
-			jaeger.SamplerOptions.SamplingServerURL(sc.SamplingServerURL),
 			jaeger.SamplerOptions.MaxOperations(sc.MaxOperations),
 			jaeger.SamplerOptions.OperationNameLateBinding(sc.OperationNameLateBinding),
 			jaeger.SamplerOptions.SamplingRefreshInterval(sc.SamplingRefreshInterval),
 		}
+		if sc.SamplerStrategiesFile != "" {
```

Comment:
nit: this logic establishes implicit priority of fields. Instead, I would suggest returning an error if both File and URL are not blanks.